modules/get-started/pages/release-notes/helm-charts.adoc

```diff
@@ -0,0 +1,40 @@
+= What's New in the Helm Charts
+:description: Summary of new features and updates in the Helm charts for Redpanda and Redpanda Console.
+
+This topic includes new content and significant changes in the Redpanda and Redpanda Console Helm charts. For a complete list of all updates, see:
+
+- https://github.com/redpanda-data/redpanda-operator/blob/main/charts/redpanda/CHANGELOG.md[Changelog for the Redpanda chart^].
```

Comment:
We'll probably want these links to be on the "stable" release branch for this version of the docs. `main` will only have changes for `25.1.x`.
```suggestion
- https://github.com/redpanda-data/redpanda-operator/blob/release/v2.4.x/charts/redpanda/CHANGELOG.md[Changelog for the Redpanda chart^].
```

I haven't cut the release branch just yet, it should be there by the end of next week.